libs/agno/agno/run/workflow.py

```diff
@@ -59,6 +59,8 @@ class BaseWorkflowRunResponseEvent:
     workflow_name: Optional[str] = None
     session_id: Optional[str] = None
     run_id: Optional[str] = None
+    step_id: Optional[str] = None
+    parent_step_id: Optional[str] = None
```

Comment:
Why are both needed? An event should just below to its step, so only `step_id` needed?

Reply:
Yes, but this is a WorkflowRunResponseEvent, or is this also the base for primitive events?